src/ontology/mondo-edit.obo

```diff
@@ -344905,8 +344907,9 @@ def: "Hemorrhagic fever with renal syndrome (HFRS) is a rodent-borne potentially
 subset: ordo_disease {source="Orphanet:340"}
 synonym: "Hantavirosis" EXACT [Orphanet:340]
 synonym: "Hantavirus fever" EXACT [Orphanet:340]
-xref: ICD10:A98.5+ {source="ORDO:340/e", source="Orphanet:340"}
-xref: ICD10:N08.0* {source="ORDO:340/e", source="Orphanet:340"}
+xref: ICD10EXP:A98.5 {source="ORDO:340/e", source="Orphanet:340"}
```

Comment:
@matentzn in this case I added the proper code - is this what I'm meant to do here?